tests/test_loggers/test_console_logger.py

```diff
@@ -140,6 +142,7 @@ def test_backtest_logging_no_tables(example_tsds: TSDataset):
 @pytest.mark.parametrize("model", [LinearPerSegmentModel(), LinearMultiSegmentModel()])
 def test_model_logging(example_tsds, model):
     """Check working of logging in fit/forecast of model."""
+    tslogger.loggers = []
```

Comment:
I think we should fix this by adding try/finally in broken tests.

Reply:
Mark the tests with "skip", this way they will not change the `tslogger`